demo/index.js

```diff
@@ -47,6 +47,16 @@ var plugin = function (config) {
 
     isRemovalOfLastAssocListItemAllowed(/*field*/) {
       return false;
+    },
+
+    createElement_Fieldset: function (props, ...children) {
```

Comment:
I wonder if maybe you want to make this conditional for a custom property on `props.field`, just to remove noise in the demo? Otherwise, the next developer may be confused why "I'm a custom fieldset!" is showing up everywhere.